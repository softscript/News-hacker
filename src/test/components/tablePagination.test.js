import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});
import HackerNewsTable from './../../components/newsDataTable';
const title = 'Home';
let wrapped = shallow(<HackerNewsTable>{title}</HackerNewsTable>);
describe('Testing Table pagination action button', () => {
    
  it('should render the Title Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the Titles children', () => { 
    // expect(wrapped.find('h1').text()).toEqual(title);
  });
});