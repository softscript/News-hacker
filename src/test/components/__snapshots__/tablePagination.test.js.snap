// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Table pagination action button should render the Title Component correctly 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Grid))
    container={true}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      sm={12}
      xs={12}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-root-1"
      >
        <WithStyles(ForwardRef(Table))
          className="makeStyles-table-2"
        >
          <WithStyles(ForwardRef(TableHead))
            className="table-head"
          >
            <WithStyles(ForwardRef(TableRow))
              className="table-head-cell"
            >
              <WithStyles(ForwardRef(TableCell))
                align="left"
              >
                Comments
              </WithStyles(ForwardRef(TableCell))>
              <WithStyles(ForwardRef(TableCell))
                align="left"
              >
                Vote Count
              </WithStyles(ForwardRef(TableCell))>
              <WithStyles(ForwardRef(TableCell))
                align="left"
              >
                Up Vote
              </WithStyles(ForwardRef(TableCell))>
              <WithStyles(ForwardRef(TableCell))
                align="left"
              >
                News Details
              </WithStyles(ForwardRef(TableCell))>
            </WithStyles(ForwardRef(TableRow))>
          </WithStyles(ForwardRef(TableHead))>
          <WithStyles(ForwardRef(TableBody)) />
        </WithStyles(ForwardRef(Table))>
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      sm={12}
      xs={12}
    >
      <Styled(MuiBox)>
        <pagination
          currentPage={0}
          totalPage={0}
        />
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    className="thin-hr"
    item={true}
    sm={12}
    xs={12}
  >
    <hr />
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    sm={12}
    xs={12}
  >
    <Styled(MuiBox)>
      <VotingChart>
        Home
      </VotingChart>
    </Styled(MuiBox)>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    className="thick-hr"
    item={true}
    sm={12}
    xs={12}
  >
    <hr />
  </WithStyles(ForwardRef(Grid))>
</Fragment>
`;
